zyte_api/aio/retry.py

```diff
@@ -75,9 +75,9 @@ class RetryFactory:
         # wait 20-40s again
         wait_fixed(20) + wait_random(0, 20),
 
-        # wait from 30 to 330s, with full jitter and exponentially
+        # wait from 30 to 630s, with full jitter and exponentially
         # increasing max wait time
-        wait_fixed(30) + wait_random_exponential(multiplier=1, max=300)
+        wait_fixed(30) + wait_random_exponential(multiplier=1, max=600)
```

Comment:
My experience with `zyte-autoextract` is that 300 can still be low and generate significant pressure in the server.  Doubling it seems safe to me. 

Imagine a project with 200 concurrent requests going on. The pressure under restrictions will be 0.5 RPS with `max=300` and 0.25 RPS with `max=600`.